docs/source/features.md

```diff
@@ -407,7 +409,7 @@ class Ord a where
     x > y = y < x
 ```
 
-## Coppaterns
+### Coppaterns in Type Class Instances
```

Comment:
This is not your fault, but it should be "copatterns" (or "co-patterns") rather than coppatterns. No cops allowed here!